src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -766,16 +786,22 @@ GenerateSizeQueryOnMultiplePlacements(List *shardIntervalList,
 			 */
 			continue;
 		}
+
+		/*
+		 * We need to build the shard relation name, being an index or table ...
+		 */
```

Comment:
nit: We prefer such single line comments that starts with a small-case, if the comment is short enough.

```suggestion
		/* we need to build the shard relation name, being an index or table */
```